PENDING_RELEASE_NOTES.md

```diff
@@ -2,6 +2,12 @@
 ## Version 14.7.1
 _Date TBD_
 
+### Table Modifications
```

Comment:
Should be in PENDING_PENDING.

Reply:
addressed

Reply:
Sorry, I didn't follow that up with "...and not in PENDING